conjure-core/src/main/java/com/palantir/conjure/parser/types/names/FieldName.java

```diff
@@ -104,8 +73,8 @@ public static FieldName of(String name) {
     }
 
     /** Converts this {@link FieldName} to a {@link FieldName} with the given case. */
-    public final FieldName toCase(Case targetCase) {
-        return FieldName.of(nameCase().convertTo(this, targetCase));
+    public final FieldName toCase(CaseConverter.Case targetCase) {
+        return FieldName.of(nameCase().convertTo(this.name(), targetCase));
```

Comment:
Need to validate this.name() is actually what we want here.

Reply:
It is - https://github.com/palantir/conjure/pull/234/files#diff-54cb02b432715a4757215e1539a57a71L67